presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcMetadata.java

```diff
@@ -173,6 +174,13 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
         return handle;
     }
 
+    @Override
+    public void createTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, boolean ignoreExisting)
```

Comment:
Why is this needed? Is this related to non null columns? Should this be extracted to separate commit?